sdks/java/io/kafka/src/test/java/org/apache/beam/sdk/io/kafka/KafkaIOTest.java

```diff
@@ -145,7 +144,7 @@
         new MockConsumer<byte[], byte[]>(offsetResetStrategy) {
           // override assign() in order to set offset limits & to save assigned partitions.
           @Override
-          public void assign(final Collection<TopicPartition> assigned) {
```

Comment:
(Ack. To be followed up in a JIRA issue that enables multi-version testing. CC: @jasonkuster @ssisk.)